front/app/components/CustomFieldsForm/CustomFieldsPage.tsx

```diff
@@ -126,6 +122,13 @@ const CustomFieldsPage = ({
   // on the success page.
   const showIdeaId = idea ? !idea.data.relationships.author?.data : false;
 
+  const formCompletionPercentage = getFormCompletionPercentage({
+    customFields,
+    formValues: methods.getValues() ?? {},
+    userIsEditing: isIdeaEditPage,
+    userIsOnLastPage: currentPageNumber === lastPageNumber,
+  });
```

Comment:
Moved this calculation from the parent into this component, because here we have access to the 'live' form values (i.e. the ones that update as you type, not just when you go to the next page)